src/debugger/server_controller.ts

```diff
@@ -95,6 +95,32 @@ export class ServerController {
 	) {
 		this.debug_data = debug_data;
 
+		if (launch_instance) {
+			let godot_path: string = utils.get_configuration("editor_path", "godot");
+			let executable_line = `"${godot_path}" --path "${project_path}" --remote-debug ${address}:${port}`;
+			if (launch_scene) {
+				let filename = "";
+				if (scene_file) {
+					filename = scene_file;
+				} else {
+					filename = window.activeTextEditor.document.fileName;
+				}
+				if (path.extname(filename).toLowerCase() === ".tscn") {
```

Comment:
How about scenes saved with `tres` `scn` or other extensions ?

Reply:
Those are a thing? =o I suppose I could just get rid of the check entirely, and if the file can't launch, then it will just fail to launch.